# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libxvmc/hotfix.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libXvMC-1.0.5/src/XvMCWrapper.c.vanilla	2010-08-12 09:49:40.606821584 +0000
+++ libXvMC-1.0.5/src/XvMCWrapper.c	2010-08-12 09:50:11.923424624 +0000
@@ -46,6 +46,7 @@
 
 #include <X11/extensions/XvMC.h>
 #include <X11/extensions/vldXvMC.h>
+#include <X11/extensions/Xvlib.h>
 #include <dlfcn.h>
 #include <stdio.h>
 #include <string.h>
